index.js

```diff
@@ -1,14 +1,96 @@
 const express = require('express');
 const helmet = require('helmet');
-
+const knex = require('knex');
+const dbConfig = require('./knexfile');
+const { zooId404 } = require('./helpers/error');
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
+const db = knex({ ...dbConfig.development });
+
 // endpoints here
 
+server.get('/api/zoos', (req, res) => {
+    db('zoos')
+        .then(zoos => {
+            res.status(200).json({ zoos });
+        })
+        .catch(err => {
+            res.status(500).json(err);
+        });
+});
+
+server.post('/api/zoos', (req, res) => {
+    db('zoos')
+        .insert(req.body)
```

Comment:
You could destructure `body` here (and elsewhere) to be explicit in what you need off of the request object:
```javascript
server.post('/api/zoos', ({ body }, res) => {
    db('zoos')
        .insert(body)
```